package/Dockerfile

```diff
@@ -104,9 +104,9 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     chmod +x /usr/bin/tini /usr/bin/telemetry /usr/bin/k3s /usr/bin/kubectl /usr/bin/channelserver && \
     mkdir -p /var/lib/rancher-data/driver-metadata
 
-ENV CATTLE_UI_VERSION 2.4.10
+ENV CATTLE_UI_VERSION 2.5.0-alpha1
 ENV CATTLE_DASHBOARD_UI_VERSION latest
```

Comment:
Don't we still need to set the 2.5.0-alpha1 tag for the UI version?

Reply:
Thought we decided on latest because it doesn't update once you start up, its always stuck there. Unless you just meant revert after this alpha is out.

Reply:
pinned